lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -1,1393 +1,1416 @@
-  nestedJourney: ADDRESS_AND_FRAUD
-  exitEvents:
-    next:
-      targetState: MITIGATION_CHECK_FRAUD_SCORE
-    enhanced-verification:
-      targetJourney: FAILED
-      targetState: FAILED
-
-MITIGATION_CHECK_FRAUD_SCORE:
-  response:
-    type: process
-    lambda: check-gpg45-score
-    lambdaInput:
-      scoreType: fraud
-      scoreThreshold: 2
-  events:
-    met:
-      targetState: MITIGATION_PP_CRI_NINO
-      checkIfDisabled:
-        hmrcKbv:
-          targetState: MITIGATION_PRE_EXPERIAN_KBV_TRANSITION_PAGE
-    unmet:
-      targetJourney: FAILED
-      targetState: FAILED
-
-# Common Mitigation states - invalid-dl/invalid-passport
-MITIGATION_PP_CRI_NINO:
-  response:
-    type: cri
-    criId: nino
-  parent: CRI_STATE
-  events:
-    next:
-      targetState: MITIGATION_CRI_HMRC_KBV
-    fail-with-no-ci:
-      targetState: MITIGATION_PRE_EXPERIAN_KBV_TRANSITION_PAGE
-
-MITIGATION_PRE_EXPERIAN_KBV_TRANSITION_PAGE:
-  response:
-    type: page
-    pageId: page-pre-experian-kbv-transition
-  events:
-    next:
-      targetState: MITIGATION_CRI_EXPERIAN_KBV
-
-MITIGATION_CRI_EXPERIAN_KBV:
-  response:
-    type: cri
-    criId: kbv
-  parent: CRI_STATE
-  events:
-    fail-with-no-ci:
-      targetJourney: INELIGIBLE
-      targetState: INELIGIBLE
-    next:
-      targetState: EVALUATE_GPG45_SCORES
-    enhanced-verification:
-      targetJourney: FAILED
-      targetState: FAILED
-
-MITIGATION_CRI_HMRC_KBV:
-  response:
-    type: cri
-    criId: hmrcKbv
-  parent: CRI_STATE
-  events:
-    fail-with-no-ci:
-      targetState: MITIGATION_PRE_EXPERIAN_KBV_TRANSITION_PAGE
-    next:
-      targetState: EVALUATE_GPG45_SCORES
-    enhanced-verification:
-      targetJourney: FAILED
-      targetState: FAILED
-
-F2F_FAILED_MITIGATION_PAGE:
-  response:
-    type: page
-    pageId: pyi-f2f-technical
-  events:
-    next:
-      targetState: MITIGATION_01_IDENTITY_START_PAGE
-    end:
-      targetState: RETURN_TO_RP
+name: New P2 Identity
+description:
+  START: >-
+    The routes a user can take to prove their identity to a medium
+    confidence level (P2).
+  ENHANCED_VERIFICATION: >-
+    The routes the user is restricted to when they need to mitigate
+    a contra-indicator from the web-based verification process.
+  ALTERNATE_DOC_PASSPORT: >-
+    The routes the user is restricted to when they need to mitigate
+    a contra-indicator from the web-based document checking process.
+  ALTERNATE_DOC_DL: >-
+    The routes the user is restricted to when they need to mitigate
+    a contra-indicator from the web-based document checking process.
+  ENHANCED_VERIFICATION_F2F_FAIL: >-
+    The routes a user is restricted to when they need to mitigate a
+    contra-indicator from the web-based verification process but have
+    tried, and failed, to mitigate it using face-to-face verification.
+  REPROVE_IDENTITY: >-
+    The route a user must take to reprove their identity after an
+    account intervention.
```

Comment:
I'm not quite sure whether we need to call out each entry point explicitly like this. I'd probably prefer if:
- The description at the journey map level covers the overall aim of the journey - i.e. in this case it's just 'The routes a user can take to prove their identity to a medium confidence level (P2).'
- If we need a detailed description of the entry points themselves, perhaps we should add an optional `description` to those states (but may not be necessary).

This probably makes it easier to format in the journey map UI as well - if this description is only a short sentence!

Reply:
Yes I did this initially but changed after finding the [docs](https://team-manual.account.gov.uk/teams/IPV-Core-team/How-IPV-Core-Works/understanding-journey-map/) - I can go back to my previous version

Reply:
Cool - happy for you to use your judgement as well.

If you do need updated copy, then Angela said she's happy to produce some (or just adapt what we've already got).

Reply:
I was in two minds really - I preferred the simplicity, but didn't want to miss any information out